### PR TITLE
Change license to Apache-2.0.

### PR DESCRIPTION
**Pull request by pranaysashank**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,30 +1,202 @@
...
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
```
Comment (harendra-kumar) [inline LICENSE]: We can remove this appendix.

File: NOTICE [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Copyright 2019 Composewell Technologies
```
Comment (harendra-kumar) [inline NOTICE:2]: See https://github.com/composewell/unicode-data/blob/master/NOTICE .  It should be the same here.

**Review by harendra-kumar — CHANGES_REQUESTED** [review]
Also change the license in source file headers.

File: LICENSE (hunk 1 above)
Comment (pranaysashank) [reply LICENSE]: the appendix is part of the license and everyone seems to be keeping it

Comment (harendra-kumar) [reply LICENSE]: We considered this in `unicode-data` and decided to remove it. Let's be consistent and do the same here.

File: src/Fusion/Plugin.hs [hunk 3]
```diff
@@ -2,8 +2,8 @@
 -- Module      : Fusion.Plugin
 -- Copyright   : (c) 2019 Composewell Technologies
 --
```
Comment (harendra-kumar) [inline src/Fusion/Plugin.hs:4]: We can remove this blank line.

**Pull request merged**